### PR TITLE
LPS-176668 Adjust clay alert styles of asset publisher and journal content

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://liferay.atlassian.net/browse/LPS-176668)

**In this PR I have adjusted the styles of asset publisher widget and journal content widget, since in this PR (https://github.com/liferay-echo/liferay-portal/pull/12788) the styles where weird**
Screenshot 2023-07-17 at 13 29 19

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view.jsp [hunk 1]
```diff
@@ -127,17 +127,15 @@ if (assetPublisherDisplayContext.isEnableTagBasedNavigation() && !assetPublisher
...
-								displayType="link"
-								label="select-a-collection-to-make-it-visible"
-								onClick="<%= portletDisplay.getURLConfigurationJS() %>"
-								small="<%= true %>"
-							/>
-						</div>
+						<liferay-ui:message key="this-application-is-not-visible-to-users-yet" />
+
+						<clay:button
+							cssClass="boerder-0 p-0"
```
Comment (sandrodw3) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view.jsp:133]: Typo here

**Review by sandrodw3 — COMMENTED** [review]
Hey @DiegoHu97 this looks much better than before 😄 I just think the button/link is a bit misaligned vertically, I think it can be fixed by adding `align-text-bottom` class to it.

**Comment by DiegoHu97** [comment]
ci:test:sf

**Pull request closed without merging**